### PR TITLE
Added the --release-notes flag to demisto-sdk

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23734

## Description
Added the --release-notes flag to demisto-sdk to get the current version release notes entries.

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/__main__.py [hunk 1]
```diff
@@ -153,6 +157,15 @@ def main(config, version):
         if last_release and cur_version != last_release:
             print_warning(f'however version {last_release} is available.\n'
                           f'You should consider upgrading via "pip3 install --upgrade demisto-sdk" command.')
+        if release_notes:
+            rn_entries = tools.get_release_note_entries(cur_version)
```
Comment (roysagi) [inline demisto_sdk/__main__.py:161]: import it straight from tools. no need for `tools.YYYY`

File: demisto_sdk/__main__.py [hunk 2]
```diff
@@ -153,6 +157,15 @@ def main(config, version):
...
+
+            if not rn_entries:
+                print_warning('\nCould not get the release notes for this version.')
+            else:
+                print('The following are the release note entries for the current version:')
+                for rn in rn_entries:
+                    print(rn)
```
Comment (roysagi) [inline demisto_sdk/__main__.py:168]: use click.echo please.

File: demisto_sdk/commands/common/tools.py [hunk 3]
```diff
@@ -1857,6 +1857,33 @@ def is_uuid(s: str) -> Optional[Match]:
...
+    Args:
+        version: The current demisto-sdk version.
+
+    Return:
+        list: A list of the release notes given from the CHANGELOG file.
+    """
+
+    changelog_file_content = get_remote_file(full_file_path='CHANGELOG.md',
+                                             return_content=True,
+                                             github_repo='demisto/demisto-sdk').decode('utf-8').split('\n')
```
Comment (roysagi) [inline demisto_sdk/commands/common/tools.py:1873]: interesting idea to take it straight from the source - let's edit the release process then to always keep the changelog file in line with the release.

File: demisto_sdk/__main__.py (hunk 1 above)
Comment (yaakovpraisler) [reply demisto_sdk/__main__.py:161]: done.

File: demisto_sdk/__main__.py (hunk 2 above)
Comment (yaakovpraisler) [reply demisto_sdk/__main__.py:168]: done.

**Review by roysagi — APPROVED** [review]

**Pull request merged**